modMcf/src/org/aion/mcf/config/CfgSync.java

```diff
@@ -72,6 +79,36 @@ public void fromXML(final XMLStreamReader sr) throws XMLStreamException {
         }
     }
 
+    private static void parseSelectedStats(List<StatsType> showStatistics, String readValue) {
+        showStatistics.clear();
+
+        String[] selected = readValue.split(",");
+
+        for (String option : selected) {
+            try {
+                showStatistics.add(StatsType.valueOf(option.toUpperCase()));
+            } catch (Exception e) {
```

Comment:
is this to ignore options that are invalid?  If so I think this should only catch IllegalArgumentException  

(according to https://docs.oracle.com/javase/7/docs/api/java/lang/Enum.html#valueOf(java.lang.Class,%20java.lang.String))

Reply:
why not include the `NullPointerException`s?

Reply:
true, wouldn't hurt to also catch NullPointerException